src/commands/bans/banned.ts

```diff
@@ -0,0 +1,23 @@
+import {Command} from "@ubccpsc310/bot-base";
+import {Client, Message} from "discord.js";
+import {getBannedWord} from "../../controllers/BanController";
+
+const banned: Command = {
+    name: "banned",
+    description: "Lists the banned word (if any)",
+    usage: "banned",
+    procedure: async (client: Client, message: Message) => {
+        const banned_word = await getBannedWord();
```

Comment:
Please don't use `snake_case`

Reply:
What are the recommended style rules for Typescript?

Reply:
You should generally follow the naming conventions that already exist in the system. In this case, it's `camelCase`.